examples/dreamcast/pvr/modifier_volume_zclip/example.c

```diff
@@ -0,0 +1,304 @@
+/* Modifier volume and Zclipping example for KOS 2.1.x
+ * by Twada
+ */
+#include <kos.h>
+#include <png/png.h>
+#include <zlib/zlib.h>
+#include "pvr_zclip.h"
+
+/* textures */
+pvr_ptr_t box_tex;
```

Comment:
```suggestion
static pvr_ptr_t box_tex;
```

Reply:
We make all globals which are only used within the C file they are declared within as being `static`.